app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -975,14 +944,6 @@ private void setTextColorbyPercentage(TextView text, int value) {
         }
```

Comment:
I'll take care of it as we discussed previously as soon as I can. As I told you earlier, I didn't want to start messing with other classes without discussing the best approach beforehand.